roles/common/tasks/main.yml

```diff
@@ -34,7 +34,10 @@
     - ingress_type | lower == 'route'
 
   - set_fact:
-      route_host: "{{ (route_host != '') | ternary(route_host,ansible_operator_meta.name + '.' + _default_ingress_domain.resources[0].spec.domain | default('')) }}"
+      route_host: "{{ ansible_operator_meta.name + '.' + _default_ingress_domain.resources[0].spec.domain | default('') }}"
+    when:
+      - route_host == ''
+      - ingress_type | lower == 'route'
```

Comment:
If the user sets a `route_host` on Pulp CR the definition will be lost, right?

Reply:
only if set as ''